src/Services/ResponseService.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace Furniture\Services;
+
+class ResponseService
+{
+    const UNEXPECTED_ERROR = 'Unexpected error';
+
+    public static function createResponse(string $message, array $data): array
```

Comment:
You could make this class do a bit more work by also passing in the HTTP status code and setting it for the response here.